config.yaml

```diff
@@ -2,27 +2,26 @@
 # See LICENSE file for licensing details.
 
 options:
-  bind-address-subnet:
+  bind-address:
     type: string
     description: |
-      The selected bind address will need to be part of this subnet.
-      If this is not set, it will use the IP address from the default interface.
+      Local IP address to bind for GTP and S1AP connection.
```

Comment:
Where is this description coming from?

Reply:
from example configs from the srslte project.
https://github.com/srsran/srsRAN/blob/master/srsenb/enb.conf.example

Reply:
Nice. How about adding something like: "the iface IP address of the srs host for reaching the core." or similar? To make it clearer for the user.